backend/test/test_czi_hosted/unit/data_common/fbs/test_matrix.py

```diff
@@ -46,7 +46,7 @@ def test_encode_DataFrame(self):
                 "d": pd.Series(["x", "y", "z", "x", "y", "z", "a", "x", "y", "z"], dtype="category"),
             }
         )
-        expected_types = ((np.ndarray, np.float32), (np.ndarray, np.int32), (np.ndarray, np.uint32), (list, None))
+        expected_types = ((np.ndarray, np.float32), (np.ndarray, np.int32), (np.ndarray, np.int32), (list, None))
```

Comment:
You have the same tuple listed twice (np.ndarray, np.int32)?

Reply:
true, but that was intentional as there are two columns which the test expects to return the int32 dtype ('b' and 'c')

Reply:
Assuming the response clarified - if not, please unresolve and add further info. TY!